examples/viewer.html

```diff
@@ -590,9 +621,29 @@
           false).start();
       addShowContainer('7');
 
-      showContainer('1');
+      new Viewer(
+            'container8',
+            '8',
+            '../test/manual/amp-carousel-0-2/vertical.amp.html?amp_js_v=0.1',
```

Comment:
Requiring every e2e test that runs in the viewer to be listed here isn't ideal. Just noting for a future refactor PR.

Reply:
+1, the block below this one references `container-dynamic` and that's where the e2e tests load into. I should probably remove this `container8` block that I made for sanity checking.